content/docs/cms.md

```diff
@@ -24,31 +23,31 @@ const cms = new TinaCMS()
 
 The `TinaCMS` constructor receives an object that can be used to configure CMS behavior. See [CMS Configuration](#cms-configuration) for details.
 
-### The &lt;Tina&gt; Component
```

Comment:
This renders as shown:

![image](https://user-images.githubusercontent.com/15221702/76535289-052a2c80-6451-11ea-8a76-a34d69ec57c6.png)